muhireheir/mybrand-1
# Ch server setup

> #### What does this PR do?
> * create nodejs server
> * connect to mongodb

> #### Description of Task to be completed
> * to create a server for my project
> #### How should this be manually tested?
>  * clone the repository
> * change to ch-server-setup branch
> * go into server folder
> * open  terminal  in server folder
> * run  the  following   codes `npm install` and wait until packages are finished to install
> * run  the  following   codes `npm start`
> * open postman
> * i  url enter the following adress `localhost:3000/home`
> * send request with any http method
> * see  the image below  to have see the expected result
> * Done.
> #### Any background context you want to provide?
> * 
> #### What are the relevant Trello story?
>  * [https://trello.com/c/qSoD6qVu](https://trello.com/c/qSoD6qVu)
> #### Screenshots (if appropriate)
![setup](https://user-images.githubusercontent.com/26524106/93498305-eb8f0380-f911-11ea-9b78-e3f7f4e6ab27.png)
